### DISCUSSION TITLE
Is this production ready to replace Ethers.js?

### DISCUSSION THREAD

**Discussion by adriandelgg** [wevm/viem#705] [Question] [answered]
As the title says, I'm wondering if this is fully ready to replace Ethers.js in production.

Why or why not?

**Comment by jxom** [comment, marked as answer]
It is.

Several orgs are using it in production right now such as: Stripe, Coinbase, Shopify, SushiSwap, Worldcoin, ZORA, Rainbow, Family, mintdotfun, and more.
